src/asm/rgbasm.5

```diff
@@ -1085,13 +1118,15 @@ ENDM
 The example above defines
 .Ql MyMacro
 as a new macro.
+String symbols are not expanded within the name of the macro.
 You may use the older syntax
 .Ql MyMacro: MACRO
 instead of
 .Ql MACRO MyMacro ,
 with a single colon
 .Ql \&:
 following the macro's name.
+.Pp
```

Comment:
Unlike numeric symbols, this doesn't mention that EQUS are expanded using the old syntax.